src/main/java/org/ow2/proactive/catalog/util/parser/WorkflowParser.java

```diff
@@ -89,6 +97,11 @@
            .forEach((name, value) -> addGenericInformationIfNotNull(keyValueMapBuilder, name, value));
         job.getUnresolvedVariables().forEach((jobVariableName,
                 jobVariable) -> addVariableIfNotNullAndModelIfNotEmpty(keyValueMapBuilder, jobVariable));
+        for (Task task : job.getTasks()) {
+            task.getVariables().forEach((taskVariableName,
+                    taskVariable) -> addDependencyIfCatalogObjectModelExist(keyValueMapBuilder, taskVariable));
+
+        }
```

Comment:
Please point me where the variables are parsed on job level (not for tasks)?

Reply:
Line 98 and 145